app/src/modules/autoUpdater.js

```diff
@@ -4,8 +4,8 @@ import {
   IPC_DOWNLOAD_UPDATE_PROGRESS,
   IPC_DOWNLOAD_UPDATE_START,
   IPC_UPDATE_AVAILABLE,
-  IPC_UPDATE_STARTED
-} from "../../../src/const/ipcGlobal";
+  IPC_UPDATE_STARTED,
+} from '../../../src/const/ipcGlobal';
 
 const getErrorMessage = (error) => {
   if (error.indexOf('404 Not Found') > -1 || error.indexOf('command is disabled') > -1) {
```

Comment:
1. Gives no error message
`wc:OSKARabe0d171b4c667c6842e9a2f81ed8aa6697e60bf4fb3602855803e2065efa03e@2?relay-protocol=iridium&symKey=ed18779a45f42a7b18497a2dc87319636b75340b3ee6f6ff9d6305f8434ab901`

2. Adding 2 extra chars in the end when trying to connect makes it crash without error message under the input
![image](https://github.com/LiskHQ/lisk-desktop/assets/8784876/6804e0c4-60f1-4ab0-a3f6-a2870c8c4005)